src/phasorpy/cursors.py

```diff
@@ -0,0 +1,299 @@
+"""Select regions of interest (cursors) from phasor coordinates.
+
+    The ``phasorpy.cursors`` module provides functions to:
+
+- create masks for regions of interests in the phasor space:
+
+  - :py:func:`mask_from_circular_cursor`
+
+- create masks for regions of interests in the polar space:
+
+  - :py:func:`mask_from_polar_cursor`
+
+- create a pseudo-color array of average signal from cursor regions:
+
+  - :py:func:`pseudo_color`
+
+"""
+
+from __future__ import annotations
+
+__all__ = [
+    'mask_from_circular_cursor',
+    'mask_from_polar_cursor',
+    'pseudo_color',
+]
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ._typing import (
+        Any,
+        ArrayLike,
+        NDArray,
+    )
+
+import numpy
+
+from phasorpy.color import CATEGORICAL
+
+
+def mask_from_circular_cursor(
+    real: ArrayLike,
+    imag: ArrayLike,
+    center_real: ArrayLike,
+    center_imag: ArrayLike,
+    /,
+    *,
+    radius: ArrayLike = 0.05,
+    axis: int = 0,
+) -> NDArray[numpy.bool_]:
+    """Return masks for circular cursors of phasor coordinates.
+
+    Parameters
+    ----------
+    real : array_like
+        Real component of phasor coordinates.
+    imag : array_like
+        Imaginary component of phasor coordinates.
+    center_real : array_like, shape (n,)
+        Real coordinates of circle centers.
+    center_imag : array_like, shape (n,)
+        Imaginary coordinates of circle centers.
+    radius : array_like, shape (n,)
+        Radii of circles.
+    axis : int, optional
+        Axis along which the masks are returned. Default is 0.
```

Comment:
Can you give an example where axis is useful?

Reply:
It's just a convenience that may be useful for someone if the mask is needed in another axis, although I know it may be done very easily outside the function. But since I though both masking function should return the mask in the same axis, and in the circular mask it was needed to explicitly move the mask axis to the first one to match the polar mask, it was little work to make it more versatile in case it was needed. But if you think this is not necessary the case, we can modify it so that it always returns the mask in the first axis.

Reply:
I still don't see the advantage to handle multiple cursors at once. But I'm OK to get this going.

Reply:
I was thinking in the case where you have a sequence of centers, it's easier to work with it directly than to have to iterate through it's elements. For example:

`lifetimes = [1.0, 8.0, 10.0]
center_real, center_imag = phasor_from_lifetime(80, lifetimes)
mask = mask_from_circular_cursor(0.5, 0.5, center_real, center_imag)`

Also, for the pseudo-color function as is implemented it accepts all masks in one array, and it's convenient for the masks function to return already all masks in one array. 

But if you believe that handling multiple cursors at once makes the function or the documentation more complex, we can discuss and go back to handle only one at a a time.